src/components/dashboard/constants.js

```diff
@@ -21,3 +22,6 @@ export const SECTION_ITEM_LIMIT = 8;
 export const USER_SUFFIX = "@iplantcollaborative.org";
```

Comment:
Minor. I use something similar in analyses too - https://github.com/cyverse-de/sonora/blob/b597a4f14a2198676576b72679dbd33c44866417/src/constants.js#L27 and https://github.com/cyverse-de/sonora/blob/b597a4f14a2198676576b72679dbd33c44866417/src/components/analyses/listing/TableView.js#L284. I guess we don't need this declared again here and just the use one from `src/constants.js` ?

Reply:
I changed the constant so it's based on the top-level `IPLANT` constant. Trying to limit inter-component dependencies, otherwise.